tests/unit/tabs/tabs_options.js

```diff
@@ -141,6 +142,12 @@ test( "{ collapsible: true }", function() {
 	element.find( ".ui-state-active .ui-tabs-anchor" ).click();
 	equal( element.tabs( "option", "active" ), false );
 	state( element, 0, 0, 0 );
+
+	element.tabs( "option", "collapsible", false );
+	ok( !element.hasClass( "ui-tabs-collapsible" ), "main element is not .ui-tabs-collapsible" );
```

Comment:
I had wondered the same but its used so infrequently in comparison i was not sure it was worth it. If we were to do this perhaps it would make sense to add a third param to the one we already have since it could be implemented without adding any lines just a couple ternary's. 

or maybe just abstract the logic into a separate function and just have `hasClasses` and `notHaveClasses` call it to make the api cleaner.